index.html

```diff
@@ -0,0 +1,298 @@
+                  <li class="h5 text-muted  mx-3">2015</li>
+                </ul>
+              </div>
+              <div><p class="lead font-weight-light">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p></div>
+            </h1>
+            <div class="btn-group mb-4" role="group" aria-label="Basic example">
+              <button type="button" class="bt-tag btn btn-sm   mx-1 text-primary ">html</button>
+              <button type="button" class="bt-tag bt-tag btn btn-sm  mx-1 text-primary">css</button>
+              <button type="button" class="bt-tag bt-tag btn btn-sm  mx-1 text-primary">ruby on rail</button>
+              <button type="button" class="bt-tag btn  btn-sm  mx-1  text-primary">javascript</button>
+            </div>
+            <div>
+              <button type="button" class="btn btn-outline-primary">See project</button>
+            </div>
+          </div>
+          <div class="col-md-5 text-center  ">
+            <img src="img/SnapshootPortfolio2.svg" class="img-fluid" alt="ebook">
+          </div>
+          
+        </div>
+      </div>
+    </section>
+    <section id="intro" class=" py-5">
+      <div class="container-lg bg-white py-5 rb ">
+        <div class="row g-4 justify-content-space-between align-items-center">
+          <div class="col-md-5 text-center  ">
+            <img src="img/SnapshootPortfolio3.svg" class="img-fluid" alt="ebook">
+          </div>
+          <div class="col-md-5 text-center text-md-start">
+            <h1>
+              <div ><h1 class="h1 font-weight-bold">Facebook 360</h1></div>
+              <div class="d-flex">
+                <span class="h5">FACEBOOK</span>
+                <ul class="d-flex">
+                  <li class="h5 text-muted mx-3">Full Stack Dev</li>
+                  <li class="h5 text-muted  mx-3">2015</li>
+                </ul>
+              </div>
+              <div><p class="lead font-weight-light">Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p></div>
+            </h1>
+            <div class="btn-group mb-4" role="group" aria-label="Basic example">
+              <button type="button" class="bt-tag btn btn-sm   mx-1 text-primary ">html</button>
+              <button type="button" class="bt-tag btn btn-sm   mx-1 text-primary ">ruby on rail</button>
+              <button type="button" class="bt-tag bt-tag btn btn-sm  mx-1 text-primary">css</button>
+              <button type="button" class="bt-tag btn  btn-sm  mx-1  text-primary">javascript</button>
+            </div>
+            <div>
+              <button type="button" class="btn btn-outline-primary">See project</button>
+            </div>
+          </div>
+          
+        </div>
+      </div>
+    </section>
+
+    <section class="my-5 about-my-self">
```

Comment:
Hi @yasinabdmahmood Great Job implementing the bootstrap project. However, the about myself section is overlapping the languages section on mobile screen.